src/SFA.DAS.AssessorService.Api.Types/Models/Apply/ApplicationResponse.cs

```diff
@@ -21,44 +22,10 @@ public class ApplicationResponse
         public string ContactName { get; set; }
         public string ContactEmail { get; set; }
 
-        public string GetApplicationType()
-        {
-            if (IsSequenceRequired(ApplyConst.ORGANISATION_SEQUENCE_NO) && IsSequenceRequired(ApplyConst.STANDARD_SEQUENCE_NO))
-            {
-                return ApplicationTypes.Combined;
-            }
-            else if (IsSequenceRequired(ApplyConst.STANDARD_SEQUENCE_NO))
-            {
-                return ApplicationTypes.Standard;
-            }
-            else if(IsSequenceRequired(ApplyConst.ORGANISATION_WITHDRAWAL_SEQUENCE_NO))
-            {
-                return ApplicationTypes.OrganisationWithdrawal;
-            }
-            else if (IsSequenceRequired(ApplyConst.STANDARD_WITHDRAWAL_SEQUENCE_NO))
-            {
-                return ApplicationTypes.StandardWithdrawal;
-            }
-
-            return string.Empty;
-        }
-
-        public bool IsSequenceRequired(int sequenceNo)
-        {
-            // a sequence cannot be considered required if it does not exist in the ApplyData
-            return ApplyData?.Sequences?.Any(x => x.SequenceNo == sequenceNo && !x.NotRequired) ?? false;
-        }
-
-        public bool IsSequenceActive(int sequenceNo)
-        {
-            // a sequence can be considered active even if it does not exist in the ApplyData, since it has not yet been submitted and is in progress.
-            return ApplyData?.Sequences?.Any(x => x.SequenceNo == sequenceNo && x.IsActive) ?? true;
-        }
-
         public bool IsWithdrawalApplication => IsStandardWithdrawalApplication || IsOrganisationWithdrawalApplication;
 
-        public bool IsStandardWithdrawalApplication => GetApplicationType() == ApplicationTypes.StandardWithdrawal;
+        public bool IsStandardWithdrawalApplication => ApplicationType == ApplicationTypes.StandardWithdrawal;
```

Comment:
We should also have  IsStandardApplication and  IsSOrganizationApplication fields

Reply:
There are properties for it - I hadn't added them here yet as I didn't have a use - there isn't an organisation application as it cant exists, there is a combined application [(organisation + financial + standard), (organisation + standard), (financial + standard)] or a standard application